install/collecting_ncn_mac_addresses.md

```diff
@@ -8,13 +8,13 @@ The Bond0 MAC0 and Bond0 MAC1 are the MAC addresses for the physical interfaces
 The Bond0 MAC0 and Bond0 MAC1 should be on the different network cards to establish redundancy for a failed network card.
 On the other hand, if the node has only a single network card, then MAC1 and MAC0 will still produce a valid configuration if they do reside on the same physical card.
```

Comment:
```suggestion
On the other hand, this is not an absolute requirement. If the node has only a single network card, then this will force `MAC1` and `MAC0` to reside on the same physical card; this will still produce a valid configuration.
```